File: ford-2.0-api-dev/ford-2.0-api-dev/FordEvents.Model/FordEventsDB/Dealership.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace FordEvents.Model.FordEventsDB
{
    public partial class Dealership
    {
        public Dealership()
        {
            QuoteForms = new HashSet<QuoteForm>();
        }

        public long Id { get; set; }
        public string Code { get; set; }
        public string Name { get; set; }
        public decimal? Lat { get; set; }
        public decimal? Long { get; set; }
        public long? ProvinceId { get; set; }
        public long? CityId { get; set; }
        public string StreetNameAndNumber { get; set; }
        public string PostalCode { get; set; }
        public string Phone1 { get; set; }
        public string Phone2 { get; set; }
        public string DealerCode { get; set; }
        public DateTime? ModifiedOn { get; set; }
        public DateTime? CreatedOn { get; set; }
        public Guid? ModifiedBy { get; set; }
        public Guid? CreatedBy { get; set; }
        public bool Deleted { get; set; }

        public virtual City City { get; set; }
        public virtual Province Province { get; set; }
        public virtual ICollection<QuoteForm> QuoteForms { get; set; }
    }
}

File: ford-2.0-app-dev/ford-2.0-app-dev/src/model/Dealership.ts
import { BaseData } from "./BaseData";

export interface Dealership extends BaseData {
    name?: string;
    provinceId?: number;
    localityId?: number;
    provinceName?: string;
    localityName?: string;
    code?: string;
    streetNameAndNumber?: string;
    postalCode?: string;
    phone1?: string;
    phone2?: string;
    dealerCode?: string;
    latitude?: number;
    longitude?: number;
}